.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,48 @@
+name: Dependabot
+on: pull_request_target
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  compile-deps:
+    name: Compile deps
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    uses: aio-libs/aiohttp/.github/workflows/sub-compile-deps.yml@master
+    secrets:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  lint:
+    name: Lint
+    needs: compile-deps
+    uses: aio-libs/aiohttp/.github/workflows/sub-lint.yml@master
+  test:
+    name: Test
+    needs: compile-deps
+    uses: aio-libs/aiohttp/.github/workflows/sub-test.yml@master
+  check:  # This job does nothing and is only used for the branch protection
```

Comment:
You missed the `needs:` definition. It'll always be green.

Reply:
Good point!